crud/cfg.lua

```diff
@@ -0,0 +1,68 @@
+---- Module for CRUD configuration.
+-- @module crud.cfg
+--
+
+local checks = require('checks')
+local errors = require('errors')
+
+local stash = require('crud.common.stash')
+local stats = require('crud.stats')
+
+local CfgError = errors.new_class('CfgError', {capture_stack = false})
+
+local cfg_module = {}
+
+local function set_defaults_if_empty(cfg)
+    if cfg.stats == nil then
+        cfg.stats = false
+    end
+
+    return cfg
+end
+
+local cfg = set_defaults_if_empty(stash.get(stash.name.cfg))
+
+--- Configure CRUD module.
+--
+-- @function __call
+--
+-- @tab self
+--
+-- @tab[opt] opts
+--
+-- @bool[opt] opts.stats
+--  Enable or disable statistics collect.
+--  Statistics are observed only on router instances.
+--
+-- @return Copy of configuration table.
```

Comment:
1. Strictly speaking, it returns the updated table itself, not a copy.
2. It is not how `box.cfg()` works, however it may be useful for chain calls. Let's keep it, interesting idea.

Reply:
1. Fixed. It seems like I've changed the behavior and forgot to change the description